gef.py

```diff
@@ -648,6 +649,15 @@ def __getattr__(self, item):
     def __int__(self):
         return self.__addr
 
+    def tcachebin(self, i):
+        # Address of entry i
+        x = int(gdb.execute("p/x $_heap()", to_string=True).split(" = ")[-1], 0) + 0x10 + 0x40 + i*8
+        # addr = dereference_as_long(x) # This doesn't work. wtf. it returns the address itself
```

Comment:
Yeah `dereference_as_long` does actually dereference. It checks to see if it can be dereferenced and then returns the address 😁

Reply:
You could just use `dereference(x)` and check for `None`

Reply:
although as you're already running a gdb command you could do it all in one go:
```python
        addr = int(gdb.execute("x/gx $_heap({})".format(0x10 + 0x40 + i*8), to_string=True).split("\t")[-1], 0)
```

Reply:
Weird. Why would a function like that be done? It seems to be only used in GlibcArena. I'll look to at the very least rename it.